knowledge_base_roadmap.md

```diff
@@ -0,0 +1,92 @@
+# Knowledge base Roadmap
+
+## Ign gazebo UI
+already there: [Understanding the GUI](https://ignitionrobotics.org/docs/citadel/gui)
+Goal: introduction to the user interface of ign gazebo
+Concepts covered
+
+* Moving and rotating entities
+* Grid setting
+* Load and save configurations
+* Components inspector
+* Entity tree
+
+## Build a robot
+
+Goal: building a robot using SDF
+Concepts covered
+
+* Create your own robot using SDF file
+* Moving joints using GUI
```

Comment:
This part is not in, right? I think this issue would address it, but it may need a lot of work: https://github.com/ignitionrobotics/ign-gazebo/issues/192